pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -323,6 +323,11 @@ protected CompletableFuture<Policies> getNamespacePoliciesAsync(NamespaceName na
                         return FutureUtil.failedFuture(new RestException(e));
                     }
                     policies.get().bundles = bundleData != null ? bundleData : policies.get().bundles;
+                    if (policies.get().is_allow_auto_update_schema == null) {
```

Comment:
Question:

Why add this part of the code?

Reply:
https://github.com/apache/pulsar/pull/14409 only adds to the `getNamespacePolicies` method, forget to add to `getNamespacePoliciesAsync `, and causes the test not to pass.